### DISCUSSION TITLE
Firefox not opening in Sandboxie

### DISCUSSION THREAD

**Discussion by BilleBarrett** [sandboxie-plus/Sandboxie#4399] [General]
After an update for Firefox, the error below started showing up
can start Firefox outside of Sandboxie and it is OK
I reinstalled Firefox 2 times
![1A-Firefox](https://github.com/user-attachments/assets/8b18cff7-7f3f-4bd8-b644-7408380727dd)

**Comment by offhub** [comment]
Delete the sandboxed Firefox files from `C:\Sandbox\user\<BoxName>\drive\C\Program Files\Mozilla Firefox`.
